pkg/quota/v1/install/update_filter.go

```diff
@@ -44,3 +52,17 @@ func DefaultUpdateFilter() func(resource schema.GroupVersionResource, oldObj, ne
 		return false
 	}
 }
+
+// hasAllocatedResourcesChanged function to compare allocated resources in container statuses
+func hasAllocatedResourcesChanged(oldPod *v1.Pod, newPod *v1.Pod) bool {
+	if len(oldPod.Status.ContainerStatuses) != len(newPod.Status.ContainerStatuses) {
+		return false
+	}
+	for i, oldStatus := range oldPod.Status.ContainerStatuses {
+		newStatus := newPod.Status.ContainerStatuses[i]
+		if !reflect.DeepEqual(oldStatus.AllocatedResources, newStatus.AllocatedResources) {
```

Comment:
`DeepEqual` is quite heavy operation. Can we do this without using it?

Reply:
fixed